### PR TITLE
Create build.yml

### PR DESCRIPTION
**Pull request by faddat**
builds it

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+        targetos: [darwin, linux]
+        include:
+          - targetos: windows
+            arch: amd64
+    name: dig ${{ matrix.arch }} for ${{ matrix.targetos }}
```
Comment (ValarDragon) [inline .github/workflows/build.yml:21]: Why is it called dig? I thought dig is a DNS utility

**Comment by ValarDragon** [comment]
Thanks for adding this, this is great! Just one question on the name of `dig`, other than that LGTM

File: .github/workflows/build.yml (hunk 1 above)
Comment (faddat) [reply .github/workflows/build.yml:21]: oops!

That is taken from another chain I am working on called dig, and yes, its binary gets confused with the dns tool often. 

Thanks!

Comment (ValarDragon) [reply .github/workflows/build.yml]: ah lol, makes sense now! Name collisions are the worst

**Review by ValarDragon — APPROVED** [review]

**Pull request merged**